ow_sim_tests/scripts/test_arm_consistency.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import os
+import actionlib
+import pandas as pd
+import tf2_ros
+import roslib
+import unittest
+import owl_msgs.msg
+import numpy as np
+import random
+import matplotlib.pyplot as plt
+import rospkg
+from mpl_toolkits.mplot3d import Axes3D
+from geometry_msgs.msg import Pose, Quaternion, Point
+from action_testing import *
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'arm_consistency'
+rospack = rospkg.RosPack()
+package_path = rospack.get_path('ow_sim_tests')
+common_dir = os.path.join(package_path, 'scripts', 'consistency_test_files')
```

Comment:
The os.getcwd() actually not working, it will lead to the .ros folder, however I found the solution:
rospack = rospkg.RosPack()
package_path = rospack.get_path('ow_sim_tests')
common_dir = os.path.join(package_path, 'scripts', 'consistency_test_files')

This code will make a directory ~/ow_sim_tests/scripts/consistency_test_files and later save all the testing results out there.